data/campaigns/Heir_To_The_Throne/scenarios/11_The_Ford_of_Abez.cfg

```diff
@@ -299,47 +298,84 @@
         [/set_variable]
     [/event]
 
-    #if the players marches over to Li'sar's keep, reinforcements appear
+    #if someone marches over to Li'sar's keep, reinforcements appear
     [event]
         name=moveto
-        [filter_second]
-            side=1
-            x=0-16
-            y=23,34
-        [/filter_second]
-        {FORD_GUARD 1 29}
-        {FORD_GUARD 1 29}
-        {FORD_GUARD 1 29}
-        {FORD_GUARD 1 29}
-        {FORD_GUARD 1 29}
-        {FORD_GUARD 1 29}
-        {FORD_GUARD 1 29}
-        {FORD_GUARD 1 29}
-        {FORD_GUARD 1 29}
+        [filter]
+            [not]
+                side=2
+            [/not]
+        [/filter]
+        [filter_condition]
+            [have_unit]
+                id="Li’sar"
+            [/have_unit]
+            [have_location]
+                x,y=$x1,$y1
+                [and]
+                    x,y=5,27
+                    radius=11
```

Comment:
Did you test that it still performs well? Note that radius= can be quite slow, i know for sure at least one place where the code was changed from radius= to use xy ranges becasue of performance issues. I cannot tell without testing whether this is important here though.